### ISSUE TITLE
Error: Invalid argument name

### ISSUE THREAD

**Issue by kaihendry** [riboseinc/terraform-aws-s3-cloudfront-website#23]
Hi, Your README example into s3.tf erros with

```
Error: Invalid argument name

  on s3.tf line 55, in module "main":
  55:     "aws.main" = "aws.main"

Argument names must not be quoted.
```

Using
```

Terraform v0.12.4
+ provider.aws v2.19.0
```

**Comment by ronaldtse** [comment]
Thank for the report @kaihendry ! I believe this is due to the incompatibility with terraform 0.12 syntax. 

@phuonghyunh can you help upgrade the module to 0.12 syntax? Make a tag so we keep what we have now for 0.11 or below. Thanks!

**Comment by phuonghuynh** [comment]
Yes

**Comment by ronaldtse** [comment]
Fixed in c17aee4582a88105109ed51fcccf6c0ef783096d.

**Comment by ronaldtse** [comment]
@kaihendry sorry for the late reply, the issue should have been fixed now.
